interview-questions.md

```diff
@@ -8,50 +8,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a placeholder in a function. An argument translates that parameters meaning into a expression.
 
-Researched answer:
+Researched answer: function add(x, y) {
+                        return x + y
+                        }
+
+                    add(2, 3) --> 5.
+                    (e.g. x = 2, y = 3) x and y are parameters and 2 and 3 are the arguments.
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: .map predefined parameters: are function(), currentValue, index, array, and thisArg; index, array, and thisArg are optional.
 
-Researched answer:
+Researched answer: .map method creates a new array for each element from a array. 
 
 3. What is the difference between map and filter?
 
-Your answer:
+Your answer: map returns output in the same length of array. filter method only return targeted items in an array and returns those items.
 
-Researched answer:
+Researched answer: Research shows that one map method is used to convert values in an array. The filter method selects certain items of an array. 
 
 4. What is iteration?
 
-Your answer:
+Your answer: iteration is a process, which searches or seeks out target items to compute.
 
-Researched answer:
+Researched answer: Iteration is an sequence of instructions being repeated until result returns true.
 
 5. What is the difference between a class and an object?
 
-Your answer:
+Your answer: A class is the blueprint of what i want to create. An object follows that blueprint, however, variations of that class can be created.  
```

Comment:
Your answers in this section look good.  I would like to see you expand more on these concepts whenever possible.  Really digging into the definitions and providing examples will help get you noticed when asked these questions in an interview.